Modules/ImportExcel.md

```diff
@@ -0,0 +1,33 @@
+# Name: ImportExcel
+
+## Description:
+PowerShell module to import/export Excel spreadsheets, without Excel Edit
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| PowerShell 4.0 | [Doug Finke](https://twitter.com/dfinke) | [Github](https://github.com/dfinke/ImportExcel) | May 2015 | powershell, excel |
+
+* On the PowerShell Gallery - [ImportExcel](https://www.powershellgallery.com/packages/ImportExcel)
+____
+
+### Shared blog posts:
+* [Introducing the PowerShell Excel Module](https://blogs.technet.microsoft.com/heyscriptingguy/2015/11/25/introducing-the-powershell-excel-module-2/)
```

Comment:
Add your name and the (b | t) before the list of posts. You can check this on any other module md file.